runtime/src/weights/frame_system.rs

```diff
@@ -18,7 +18,7 @@
 //! Autogenerated weights for frame_system
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2022-06-27, STEPS: `1`, REPEAT: 1, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! DATE: 2022-08-12, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
```

Comment:
We don't need to update those weights now, so I suggest removing any changes inside `runtime/src/weights` from this PR for clarity.

Reply:
Previous review comment not addressed